Server/src/bgp/linkstate/MPLinkStateAttr.cpp

```diff
@@ -505,13 +592,49 @@ namespace bgp_msg {
                 break;
             }
             case ATTR_PREFIX_OSPF_FWD_ADDR:
-                //SELF_DEBUG("%s: bgp-ls: parsing prefix OSPF forwarding address attribute", peer_addr.c_str());
+                // SELF_DEBUG("%s: bgp-ls: parsing prefix OSPF forwarding address attribute", peer_addr.c_str());
                 LOG_INFO("%s: bgp-ls: prefix OSPF forwarding address attribute, not yet implemented", peer_addr.c_str());
                 break;
 
             case ATTR_PREFIX_OPAQUE_PREFIX:
                 LOG_INFO("%s: bgp-ls: opaque prefix attribute (len=%d), not yet implemented", peer_addr.c_str(), len);
                 break;
+                
+            case ATTR_PREFIX_SID_TLV: {
```

Comment:
We will need to add PREFIX-SID bgp attribute as well, but apparently it's not the same.   If we can reuse this code, then lets do that when adding prefix-sid BGP attribute decoding.   See 
 https://tools.ietf.org/html/draft-ietf-idr-bgp-prefix-sid-03 for details on that.